packages/raystack/components/toast/toast.module.css

```
@@ -1,3 +1,284 @@
-.toast-wrapper {
-  margin-right: var(--rs-space-3);
+/* ===== Viewport ===== */
+.viewport {
+  --gap: 0.75rem;
+  position: fixed;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)